python/MOCK_sensors.py

```diff
@@ -185,11 +190,11 @@ def breakWindSensors(self, data, brokenWindSensorStatus):
 
             if(self.brokenSensorData is None):
 
-                if np.random.randint(1, high=3) < 2:
-                    # multiplier will be between 1/MAX_WINDSPEED_MULTIPLIER_ERROR and 1
+                if np.random.randint(0, 2):
+                    # 50% chance multiplier will be between 1/MAX_WINDSPEED_MULTIPLIER_ERROR and 1
                     multiplier = 1.0 - np.random.ranf() * (1.0 - 1.0 / MAX_WINDSPEED_MULTIPLIER_ERROR)
                 else:
-                    # multiplier will be between 1 and MAX_WINDSPEED_MULTIPLIER_ERROR
+                    # 50% chance multiplier will be between 1 and MAX_WINDSPEED_MULTIPLIER_ERROR
                     multiplier = 1.0 + np.random.ranf() * (MAX_WINDSPEED_MULTIPLIER_ERROR - 1.0)
```

Comment:
Here is an example to show the behaviour I'm going for:

Suppose `MAX_WINDSPEED_MULTIPLIER_ERROR` is set to 1.2. I want the multiplier to be between 1/1.2 = 0.833 and 1.2. The first case is `1 - (a number between 0 and 1 - 1/1.2 = 0.167)`, which gives us a multiplier between 0.833 and 1. The second case is `1 + (a number between 0 and 0.2)`, which gives us a multiplier between 1 and 1.2. Together, these two possibilities give us the multiplier between 0.833 and 1.2 as desired.

It would have been more simple to read if the second case just said `muliplier = random.random()*MAX_WINDSPEED_MULTIPLIER_ERROR`, but that leaves the possibility of `random.random()` being close to zero which would lead to the multiplier being outside of the desired range from 0.833 to 1.2.

I could have instead chosen to create a multiplier between `1 - MAX_WINDSPEED_MULTIPLIER_ERROR` and `1 + MAX_WINDSPEED_MULTIPLIER_ERROR`. However, if `MAX_WINDSPEED_MULTIPLIER_ERROR` were set to 1, this would create a multiplier between 0 and 2, and reading 0 all the time seems like a much more severe error than reading double the true windspeed.